src/server/lib/email.js

```diff
@@ -1,54 +1,35 @@
-const AWS = require('aws-sdk')
+/*
+--------------------------------------------------------------------------------
+-                                 lib/email.js
+--------------------------------------------------------------------------------
 
-const ses = new AWS.SES({ region: process.env.SES_REGION })
-const fromEmail = process.env.NOTIFICATIONS_EMAIL
-const expiryMinutes = parseInt(process.env.LOGIN_EXPIRY_MINUTES) || 30
+*/
+/* eslint camelcase: 0 */
 
-function sendPasscode (email, passcode, httpOrigin) {
-  const params = {
-    Destination: {
-      ToAddresses: [email]
-    },
-    Source: fromEmail,
-    Message: {
-      Subject: {
-        Charset: 'UTF-8',
-        Data: 'CARES Reporter Access Link'
-      },
-      Body: {
-        Html: {
-          Charset: 'UTF-8',
-          Data: `<p>Your link to access the CARES Reporter is
-     <a href="${httpOrigin}/api/sessions?passcode=${passcode}">${httpOrigin}/api/sessions/?passcode=${passcode}</a>.
-     It expires in ${expiryMinutes} minutes</p>`
-        }
-      }
-    }
-  }
-  return ses.sendEmail(params).promise()
+const aws = require('./email-aws')
+const nodemailer = require('./email-nodemailer')
+const transport = process.env.AWS_ACCESS_KEY_ID ? aws : nodemailer
```

Comment:
It's a small thing, but if this is the only place you are using aws and nodemailer, you don't really need to include both modules:

const transport = process.env.AWS_ACCESS_KEY_ID ? require('./email-aws') : require('./email-nodemailer')

Reply:
Good point! Thanks. Changed.